### PR TITLE
Add Note: CephsFS ShallowCopy not supported for datamover restore

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**

### CODE REVIEW

File: docs/examples/datamover_advanced_voloptions.md [hunk 1]
```diff
@@ -360,6 +360,10 @@ spec:
             storageClassName: ocs-storagecluster-cephfs-shallow
           destinationVolumeOptions:
             moverSecurityContext: true
+            accessMode: ReadOnlyMany
```
Comment (kaovilai) [inline docs/examples/datamover_advanced_voloptions.md:363]: Does this work on ceph? It doesn't for gp2.

Comment (shubham-pampattiwar) [reply docs/examples/datamover_advanced_voloptions.md:363]: https://github.com/ceph/ceph-csi/blob/devel/docs/cephfs-snapshot-backed-volumes.md

**Review by sseago — APPROVED** [review]

**Review by eemcmullan — APPROVED** [review]

**Pull request merged**